content/_index.md

```diff
@@ -3,21 +3,39 @@ title: "Home"
 date: 2021-10-01T09:28:27-05:00
 draft: false
 ---
-# LaunchCode
-You just created a brand new curriculum! Feel free to create new chapters or segments however you normally would. You can also find useful links below that may help inspire.
-You pulled in three LaunchCode created github submodules:
-- `layouts`: provides slides shortcode and overwrites menu footer
-- `static`: provides launchcode logo, favicon, and theme variants
-- `archetypes`: provides segment archetype for new chapters
-Example usage of segment:
-```bash
-hugo new --kind segment chapter-name
-```
-Upon running the above command it will create a new chapter named `chapter-name`
-{{% notice warning %}}
-Replace the content of this entire file with the desired homepage contents of this course!
-{{% /notice %}}
-## Helpful Links
-[LaunchCode Technical Training Github](https://github.com/LaunchCodeTechnicalTraining)
-[Relearn Documentation](https://mcshelby.github.io/hugo-theme-relearn/)
-[Live link to Linux Curriculum](https://lctt-linux.netlify.app/)
+
+# C# Web Development
+
+## What is this Course
+
+This course is an introduction to object-oriented programming in C#. Students will then move on to building web applications with the industry-grade framework, ASP.NET Core MVC.
+
+## Why take this Course
+
+This course is an introduction to **C#** and the .NET framework. C# is a programming language developed by Microsoft and used in almost all of its products.
+
+This course is designed for learners who are already familiar with at least one programming language. Thus, we’ll move quickly through the syntax rules and procedural basics of the language (ie, how does one write a `for` loop in C#? a conditional statement? Etc.).
+
+As you’ve likely heard before, once you grasp the fundamentals of programming in one language, learning another becomes much easier. Professional software developers often work in environments involving several programming languages so it’s wise to start learning new ones early on in your career.
+
+Beyond the basics of programming in C#, you’ll learn some key object oriented concepts to enhance the capabilities of your programs. You’ll also download a program called **Visual Studio IDE** to get familiar with the tools of the C# developer. In the final lessons of this course, you will use a C# framework called **ASP.NET Core MVC**. This framework gives us the scaffolding necessary to create MVC programs (another concept we’ll cover) effectively and efficiently.
+
+## Where this Course Leads
+
+Upon completing the content within this course you will be capable of building a web application with ASP.NET Core MVC that can store data in a local database. 
+
+After completing this course, you are ready to begin our Liftoff program.
```

Comment:
Do students automatically enter LiftOff or is it opt in?  Might be nice to give a brief description about what they will learn in liftoff.

After completing this course is restating the sentence above -> You may continue your learning journey...

Reply:
Checked with Lin to make sure new language is appropriate but it sounds like it will be automatic in the future.

Reply:
Looks good Sally, thanks for updating.